checkstyle-tester/diff.groovy

```diff
@@ -185,8 +188,16 @@ def generateCheckstyleReport(cfg) {
     def checkstyleVersion = getCheckstyleVersionFromPomXml("$cfg.localGitRepo/pom.xml", 'version')
 
     executeCmd("mvn -e --batch-mode -Pno-validations clean install", cfg.localGitRepo)
-    executeCmd("""groovy launch.groovy --listOfProjects $cfg.listOfProjects
-            --config $cfg.checkstyleCfg --ignoreExceptions --ignoreExcludes --checkstyleVersion $checkstyleVersion""")
+
+    command = """groovy launch.groovy --listOfProjects $cfg.listOfProjects
+            --config $cfg.checkstyleCfg --ignoreExceptions --ignoreExcludes
+            --checkstyleVersion $checkstyleVersion"""
+
+    if (cfg.extraMvnRegressionOptions == true) {
```

Comment:
Strangely, this is required, since `if (cfg.extraMvnRegressionOptions)` does not work as intended. I assume this is because groovy is not strongly typed.

Reply:
Why wasn't this an issue with launch? https://github.com/checkstyle/contribution/blob/master/checkstyle-tester/launch.groovy#L243-L244

Reply:
The only difference that I can see is that we are accessing `cfg`'s fields and opposed to actually passing parameters to .`generateCheckstyleReport()`?  Should I investigate?

Reply:
Its not that big a deal.